data/core/units/dunefolk/Wyvern_Rider.cfg

```diff
@@ -14,7 +14,7 @@
     {AMLA_DEFAULT}
     cost=102
     usage=scout
-    description= _ "Although generally peaceful, wyverns can easily become fierce and angry beasts when provoked. When truly enraged they can even tear apart experienced warriors with relative ease. Most travelers would run or fight in their mighty presence, yet a few are able to tame these wild creatures. These Dunefolk riders spend years, even decades of their lives bonding with them, slowly building up their trust until they have gained true mastery over the skies. The most experienced of these Wyvern riders have ascended to be the greatest and most revered travelers of the land. While they do not usually use their knowledge for warfare, they can be threatening warriors when defending their homes."
+    description= _ "Although generally peaceful, wyverns can easily become fierce and angry beasts when provoked. When truly enraged they can even tear apart experienced warriors with relative ease. Most travelers would run or fight in their mighty presence, yet a few are able to tame these wild creatures. These Dunefolk riders spend years, even decades of their lives bonding with them, slowly building up their trust until they have gained true mastery over the skies. The most experienced of these Wyvern Riders have ascended to be the greatest and most revered travelers of the land. While they do not usually use their knowledge for warfare, they can be threatening warriors when defending their homes."
```

Comment:
Should the separate instances of wyverns and riders first mentioned also be capitalised? At least the riders should be, since they are the subject of this entry.